index.md

```diff
@@ -45,7 +47,7 @@ layout: page
   </div>
 </div>
 
-<hr>
+<div><hr></div>
```

Comment:
I agree...awkward, but I haven't found any way of telling Jekyll not to wrap it. I could remove the hr and use some inline style in the div (or just in the preceding div) and give it a bottom border perhaps?